Gruntfile.js

```diff
@@ -618,46 +663,48 @@ module.exports = function (grunt) {
             grunt.fail.fatal(report.missingFiles + " out of " + report.expectedFiles + " expected files not found");
         } else {
-            grunt.log.subhead("Verification passed".green);
+            grunt.log.ok("Verification passed".green);
             grunt.log.oklns("All expected files were present");
         }
     };
 
-    grunt.registerTask("verifyDistJS", "Verifies that the expected /dist/*.js files and their source maps were created", function () {
-        grunt.log.subhead("Verifying that expected distribution JS files are present in /dist directory");
-        var expectedFilenames = [];
-        var distributions = grunt.config.get("distributions");
-        _.forEach(distributions, function (value, distribution) {
-            var jsFilename = "infusion-" + distribution + ".js";
-            var mapFilename = jsFilename + ".map";
-            expectedFilenames.push(jsFilename, mapFilename);
-        });
-
-        var report = verifyFiles("dist", expectedFilenames);
-
+    /** Common function for use by verifyDistFiles task
+     *
+     *
+     * @param {String} message - message to be displayed when running the verify files tasks
+     * @param {Array} expectedFilenames - array of expected file names
+     * @param {String} fileDir - base directory expected to contain files
+     */
+    var verifyFilesTaskFunc = function (message, expectedFilenames) {
+        grunt.log.subhead(message);
+        var report = verifyFiles(expectedFilenames);
         displayVerifyFilesReport(report);
         processVerifyFilesReport(report);
-    });
 
-    grunt.registerTask("verifyDistCSS", "Verifies that the expected /dist/ CSS files were created", function () {
-        grunt.log.subhead("Verifying that expected distribution CSS files are present in /dist/assets directory");
-        var expectedFilenames = [];
-        var preferencesStylusFiles = grunt.file.expand("src/framework/preferences/css/stylus/*.styl");
-        _.forEach(preferencesStylusFiles, function (stylusFile) {
-            var cssFilename = stylusFile.replace(".styl", ".css");
-            var minifiedCSSFilename = stylusFile.replace(".styl", ".min.css");
-            // Remove /stylus from the path, since dist/assets won't have it
-            expectedFilenames.push(cssFilename.replace("/stylus", ""), minifiedCSSFilename.replace("/stylus", ""));
-        });
+    };
 
-        var report = verifyFiles("dist/assets", expectedFilenames);
+    grunt.registerMultiTask("verifyDistFiles", "Verify distribution files", function () {
+        var message = "Verifying all \"" + this.target + "\" files are in /dist";
+        var expectedFiles = [];
 
-        displayVerifyFilesReport(report);
-        processVerifyFilesReport(report);
+        // If the target uses the standard Grunt file options
+        if (this.files.length > 1) {
+            _.forEach(this.files, function (file) {
+                expectedFiles.push(file.dest);
+            });
```

Comment:
Minor change, but I think you can just use either [_.map](https://lodash.com/docs/4.17.15#map) or JavaScript's array [map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) method.